src/components/settings/Users/EditUser.vue

```diff
@@ -0,0 +1,219 @@
+                />
+            </label>
+
+            <button class="btn btn-primary" type="submit" :disabled="saving">
+                <span v-show="saving" class="spinner-border spinner-border-sm" role="status" aria-hidden="true" />
+                {{ $t("Update Username") }}
+            </button>
+        </form>
+
+        <h5 class="mt-5 mb-4 settings-subheading">{{ $t("Change Password") }}</h5>
+        <form class="mb-3" @submit.prevent="savePassword">
+            <div class="mb-3">
+                <label class="form-label d-block">
+                    {{ $t("Current Password") }}
+                    <input
+                        v-model="currentPassword"
+                        type="password"
+                        class="form-control mt-2"
+                        required
+                        :disabled="savingPassword"
+                    />
+                </label>
+            </div>
+
+            <div class="mb-3">
+                <label class="form-label d-block">
+                    {{ $t("New Password") }}
+                    <input
+                        v-model="newPassword"
+                        type="password"
+                        class="form-control mt-2"
+                        required
+                        :disabled="savingPassword"
+                    />
+                </label>
+            </div>
+
+            <div class="mb-3">
+                <label class="form-label d-block">
+                    {{ $t("Repeat New Password") }}
+                    <input
+                        v-model="repeatNewPassword"
+                        type="password"
+                        class="form-control mt-2"
+                        required
+                        :disabled="savingPassword"
+                    />
+                </label>
+            </div>
+
+            <button class="btn btn-primary" type="submit" :disabled="savingPassword">
+                <span v-show="savingPassword" class="spinner-border spinner-border-sm" role="status" aria-hidden="true" />
+                {{ $t("Update Password") }}
+            </button>
+        </form>
+
+        <h5 class="mt-5 mb-4 settings-subheading">{{ $t("Permissions") }}</h5>
+        <div class="form-check form-switch">
+            <label class="form-check-label">
+                <input
+                    :checked="active"
+                    class="form-check-input"
+                    style="scale: 1.4; cursor:pointer"
+                    type="checkbox"
+                    :disabled="saving"
+                    @click="debounce(() => { active = !active; save({ active }); })"
+                >
+                <div class="ps-2">{{ $t("Active") }}</div>
+            </label>
+        </div>
+    </template>
+</template>
+
+<script>
+import { useToast } from "vue-toastification";
+const toast = useToast();
+
+/**
+ * Get debounced function
+ * @returns {function} debounced function
+ */
+function Debounce() {
```

Comment:
I'm not aware of "other usage", but it's a very generic helper so feel free to tell me what to do with it?

Reply:
I am refering to this code:

https://github.com/M1CK431/uptime-kuma/blob/2b2a352de69f220d10f087a90be864a0f014e527/src/mixins/socket.js#L766-L771

Reply:
As you stand yourself, I have to keep my PR atomic and so changing that code should be done in another PR I guess. But, perhaps I could move my helper in a global helpers file and so, when some people will want to refactor that code to benefit from this new helper it will be easier to do... wdyt?

Reply:
Fair point.
Moving this to a helper file would be beneficial in this case, agree

Reply:
Moved in `util-frontend.js`.